### PR TITLE
fix checkdeps.py

### PR DESCRIPTION
**Pull request by bridiver**
Resolves https://github.com/brave/brave-browser/issues/14024

## Submitter Checklist:

- [x] There is a [ticket](https://github.com/brave/brave-browser/issues) for my issue.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [x] Wrote a good [PR/commit description](https://google.github.io/eng-practices/review/developer/cl-descriptions.html)
- [x] Added appropriate labels (`QA/Yes` or `QA/No`; `release-notes/include` or `release-notes/exclude`; `OS/...`) to the associated issue
- [x] Checked the PR locally: `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests`, `npm run lint`, `npm run gn_check`, `npm run tslint`
- [x] Ran `git rebase master` (if needed).
- [ ] Requested a security/privacy review as needed.

## Reviewer Checklist:

- [ ] New files have MPL-2.0 license header.
- [ ] Adequate test coverage exists to prevent regressions
- [ ] Major classes, functions and non-trivial code blocks are well-commented
- [ ] Changes in component dependencies are properly reflected in `gn`
- [ ] Code follows the [style guide](https://chromium.googlesource.com/chromium/src/+/HEAD/styleguide/c++/c++.md)
- [ ] Test plan is specified in PR before merging

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on.
- [ ] All relevant documentation has been updated, for instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protections
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-Compatibility-Exceptions-in-Brave
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide
  - [ ] https://github.com/brave/brave-browser/wiki/P3A

### CODE REVIEW

**Comment by bridiver** [comment]
There are a large number of code owners on this PR, but this change is not relevant to them and we should try to reduce the number of blockers for entire directories

**Review by bbondy — APPROVED** [review]

**Review by zenparsing — APPROVED** [review]
Thanks! I'm looking forward to using DEPS files in the future to layer subcomponents more appropriately in `bat-native-ledger`.

**Comment by bridiver** [comment]
added skip labels to re-run windows only for powershell commandline fix

File: vendor/bat-native-ledger/src/DEPS [hunk 1]
```diff
@@ -0,0 +1,3 @@
+include_rules = [
+ # "+rapidjson",
```
Comment (tmancey) [inline vendor/bat-native-ledger/src/DEPS:2]: nit: commented out DEP, is this still required?

**Review by tmancey — APPROVED** [review]
LGTM++

**Comment by tmancey** [comment]
@bridiver Raised https://github.com/brave/brave-browser/issues/14028 to resolve DEPS for Brave Ads

**Review by kylehickinson — APPROVED** [review]
Is there documentation around writing these DEPS files? (like `include_rules` vs `specific_include_rules`, where to put these files, +/-, etc.)

**Review by AlexeyBarabash — APPROVED** [review]

File: vendor/bat-native-ledger/src/DEPS (hunk 1 above)
Comment (bridiver) [reply vendor/bat-native-ledger/src/DEPS:2]: no, it was moved further down and I forgot to remove it

Comment (bridiver) [reply vendor/bat-native-ledger/src/DEPS:2]: I'll remove after the CI run finishes to I don't interrupt it

**Comment by bridiver** [comment]
> Is there documentation around writing these DEPS files? (like `include_rules` vs `specific_include_rules`, where to put these files, +/-, etc.)

https://chromium.googlesource.com/chromium/src/+/master/buildtools/checkdeps/README.md

but it should be uncommon to have to change them once they are stable because what is allowed to be included is generally pretty stable

**Review by iefremov — APPROVED** [review]

**Review by mkarolin — APPROVED** [review]
++

**Pull request merged**